build.sh

```diff
@@ -14,4 +14,4 @@ else
     die
 fi
 
-(cd src && ./build.sh) || die
+(cd src && ./build.sh $@) || die
```

Comment:
Experiments with bash suggest you need double quotes around that `$@`.

Reply:
hrm, wfm.

Reply:
ok discussed; feel free to fix it in followup, I don't want to wait another hour for CI..